assets/stylesheets/kitten/components/buttons/_button.scss

```diff
@@ -312,4 +312,8 @@
       width: auto;
     }
   }
+
+  .is-readonly {
```

Comment:
Je spécifierais `.k-Button.is-readonly` pour ne pas rendre la classe trop générique sur l'app.